src/fmu/dataio/datastructure/meta/meta.py

```diff
@@ -341,23 +398,25 @@ class Context(BaseModel):
     stage: enums.FmuContext
 
 
-class FMUClassMetaCase(BaseModel):
+class FMUCaseAttributes(BaseModel):
     """
-    The FMU block records properties that are specific to FMU
+    The ``fmu`` block contains all attributes specific to FMU. The idea is that the FMU results
+    data model can be applied to data from *other* sources - in which the fmu-specific stuff
+    may not make sense or be applicable.
     """
 
-    case: FMUCase
-    model: FMUModel
+    case: Case
+    model: Model
 
 
-class FMUClassMetaData(BaseModel):
+class FMUAttributes(FMUCaseAttributes):
```

Comment:
`FMUAttributes` is derived from the simple FMUCaseAttributes, where case of course responds to the reduced metadata included in just case metadata

Reply:
Seems logical

Reply:
"of course"... :)